src/service/video-manager-impl.js

```diff
@@ -755,11 +876,64 @@ class VideoEntry {
    *
    * @private
    */
-  unDockVideo_() {
+  finishDocking_() {
     // Restore the video inline
     this.internalElement_.classList.remove(DOCK_CLASS);
     this.internalElement_.setAttribute('style', '');
+    this.dockingState_ = DockingStates.INLINE;
     this.video.showControls();
```

Comment:
why? if video is still playing at the time of undock, I don't think we should force show controls (given controls hide away when videos are playing in most players). Controls would be one tap away anyway. If video was not playing at undock time, then sure, we should show them.